src/CSET/operators/constraints.py

```diff
@@ -181,6 +181,39 @@ def generate_time_constraint(
     return time_constraint
 
 
+def generate_area_constraint(
+    lat_start: float, lat_end: float, lon_start: float, lon_end: float, **kwargs
+) -> iris.Constraint:
+    """Generate an area constraint between latitude/longitude limits.
+
+    Operator that takes a set of latitude and longitude limits and returns a
+    constraint that selects grid values only inside that area.
+
```

Comment:
Add a note that it works with the grid's own lat/lon, so it is affected by rotated pole grids.

Reply:
Done!